.azure/scripts/setup-kubernetes.sh

```diff
@@ -4,6 +4,8 @@ set -xe
 rm -rf ~/.kube
 
 KUBE_VERSION=${KUBE_VERSION:-1.19.0}
+MINIKUBE_REGISTRY_IMAGE=${REGISTRY_IMAGE:-"registry"}
+MINIKUBE_REGISTRY_IMAGE_S390X=${REGISTRY_IMAGE_S390X:-"s390x/registry:2.8.0-beta.1"}
```

Comment:
Stupid question ... why do we need two options? Can't the `MINIKUBE_REGISTRY_IMAGE` be used to set the S390x version of the registry when running on s390x?

Reply:
Good point. I will change it.

Reply:
Done